### PR TITLE
Add recipe for mygeotab

### PR DESCRIPTION
**Pull request by parthea**
For https://pypi.python.org/pypi/mygeotab

### CODE REVIEW

**Comment by parthea** [comment]
@aaront Please confirm you're ok being listed as a maintainer here

**Comment by aaront** [comment]
Thanks @parthea!

File: recipes/mygeotab/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  license_file: LICENSE
+  summary: 'An unofficial Python client for the MyGeotab API'
+  description: An Apache2 Licensed, unofficial Python client for the MyGeotab SDK.
+  doc_url: http://mygeotab-python.readthedocs.io/en/latest/
+  dev_url: https://github.com/Geotab/mygeotab-python
+
+extra:
+  recipe-maintainers:
+    - parthea
+    - aaront
```
Comment (synapticarbors) [inline recipes/mygeotab/meta.yaml]: @aaront -- Are you ok being added as a co-maintainer of the `mygeotab` recipe on conda-forge? You were added by @parthea.

File: recipes/mygeotab/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
```
Comment (synapticarbors) [inline recipes/mygeotab/meta.yaml]: It looks like there is an `entry_point` defined in the `setup.py`:

https://github.com/Geotab/mygeotab-python/blob/master/setup.py#L47-L49

Please add the entry point to the `build` section as described here:
https://conda.io/docs/building/meta-yaml.html#python-entry-points

and add a test in the testing section as shown here:
https://conda.io/docs/building/meta-yaml.html#test-commands

File: recipes/mygeotab/meta.yaml (hunk 1 above)
Comment (aaront) [inline recipes/mygeotab/meta.yaml]: @synapticarbors yeah

**Comment by synapticarbors** [comment]
It looks like this is failing because of python 3.5+ syntax in the async `ext`. In python 2.7 the binary .pyc files are attempting to be created but there is unrecognized `async def` syntax. I'm not exactly sure how to deal with this. There was a similar issue mentioned in the jinja2 issues:

https://github.com/pallets/jinja/issues/653

although there isn't a good solution that comes out of it. 

@conda-forge/staged-recipes - Has anyone seen this issue before and know of the proper workaround?

Here is the error on CircleCI:
https://circleci.com/gh/conda-forge/staged-recipes/17041?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

**Comment by aaront** [comment]
@synapticarbors I'm going to take a look at this upstream and do a bit of a refactor

**Comment by parthea** [comment]
This is not ready for review yet. I'm going to close this PR and re-open once its ready.

**Pull request closed without merging**